nodes/persistence-node/src/utils/isTextFile.ts

```diff
@@ -0,0 +1,5 @@
+const textExtensions = [".graphql", ".yaml"]
```

Comment:
@nerfZael Should we cover other extensions here, or is it expected that wrappers will only contain those two?